### ISSUE TITLE
Alias support for commonly used commands

### ISSUE THREAD

**Issue by james2doyle** [Nukesor/pueue#99]
**Is your feature request related to a problem? Please describe.**
I have a some common commands that I run a few times a week that take a while to run. Writing out the commands for pueue can be tedious and depending on how long its been, they may not be easy to find in my history.

**Describe the solution you'd like**
Be able to set aliases in the pueue config and easily reference them from the cli

**Describe alternatives you've considered**
Making functions in my shell profile to wrap up `pueue add ...$args`. Making scripts in `~/.config` beside the pueue yaml config and then referencing them from the cli

**Additional context**

Some of these examples are pretty contrived but I think the idea is sound:

```diff
---
+aliases:
+  big-script: "sh ~/Projects/project/big-script.sh"
+  brew: "brew update && brew upgrade && brew cleanup"
+  cinstall: "cargo install --path $PWD"
+  ndeploy: "npm run build && npm run deploy"
+  server-sync: "rsync --update --archive --delete --recursive --compress --progress . $SERVER_PUBLIC_DIR/"

client:
  // etc, etc, etc...
```

```sh
pueue run brew
```

**Comment by Nukesor** [comment]
I'm confused.
Do you not know about the aliasing feature yet (recently released in v0.6.0).
https://github.com/Nukesor/pueue#aliases

Or do you propose a new feature for aliases?

**Comment by james2doyle** [comment]
Haha wow I totally missed that!

My bad. I will close. Obviously thinking ahead haha
